packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/dashboard-component.spec.tsx

```diff
@@ -60,7 +47,15 @@ export const mocked_props = {
 
 describe('<Dashboard />', () => {
     it('user guide button click should be enabled', () => {
```

Comment:
i guess this test could be in the test file of `user-guide.spec.tsx`, right? it would be logical...

Reply:
same here this is the main dashboard component file this was legacy we don't need this test case will remove it 👍

Reply:
Since you refactor the tests you can move that test of <UserGuide to required file `user-guide.spec.tsx` instead of save test of <UserGuide  component in <Dashboard component.

Reply:
yes we already have a useguide test case spec file i will have to delete this file thats y i din want to remove it